uamqp/async_ops/client_async.py

```diff
@@ -807,6 +807,7 @@ def __init__(
                 properties=self._link_properties,
                 error_policy=self._error_policy,
                 encoding=self._encoding,
+                desired_capabilities=self._desired_capabilities,
```

Comment:
Does this need to be passed into the MessageReceiver constructor?

Reply:
I'm not sure about this, I see `desired_capabilities` appears in the `ATTACH` frame described in the amqp protocol.
And I see fields like `rcv-settle-mode` are being set in the constructor, so I put it here..

I'm neural on this and okay to remove it from the constructor.
What's your thought?

Reply:
Nevermind - I misread the diff - this is fine I think